### PR TITLE
Update react-without-jsx.md

### PR DESCRIPTION
**Pull request by RichardOtvos**
The Hungarian translation of the react-without-jsx.md page

### CODE REVIEW

File: content/docs/react-without-jsx.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 id: react-without-jsx
-title: React Without JSX
+title: React JSX nélkül
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+Nem kötelező a JSXet választani, ha Reactot használsz. A React JSX nélkül különösen hasznos, ha nem akarsz külön fordítási lépést létrehoznod a build környezetedben.
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:7]:
```suggestion
Nem kötelező a JSX-et választani, ha Reactet használsz. A React JSX nélkül különösen hasznos, ha nem akarsz külön fordítási lépést létrehozni a build környezetedben.
```

File: content/docs/react-without-jsx.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-Each JSX element is just syntactic sugar for calling `React.createElement(component, props, ...children)`. So, anything you can do with JSX can also be done with just plain JavaScript.
+Minden egyes JSX elem csak egy rövidítés a `React.createElement(component, props, ...children)` metódus hívására. Így, mindent amit JSXel le tudsz írni, meg tudod csinálni JSX nélkül, síma JavaScriptet használva is.
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:9]:
```suggestion
Minden egyes JSX elem csak egy rövidítés a `React.createElement(component, props, ...children)` metódus hívására. Így, mindent amit JSX-szel le tudsz írni, meg tudod csinálni JSX nélkül is, sima JavaScriptet használva.
```

File: content/docs/react-without-jsx.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 
-For example, this code written with JSX:
+Például, ez a JSXet használó kódrészlet
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:11]:
```suggestion
Például, ez a JSX-szet használó kódrészlet:
```

File: content/docs/react-without-jsx.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ ReactDOM.render(
 );
 ```
 
-can be compiled to this code that does not use JSX:
+átírható a következő, JSXet nem használó kódra:
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:26]:
```suggestion
átírható a következő, JSX-szet nem használó kódra:
```

File: content/docs/react-without-jsx.md [hunk 5]
```diff
@@ -37,12 +37,11 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
+Próbáld ki [az online Babel fordítót](babel://jsx-simple-example), ha szeretnél több példát látni rá, hogyan alakítjuk át a JSXet JavaScriptté.
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:40]: It looks like there is a line misalignment from here downwards. Please fix that, so it looks exactly like the original (English) text.

File: content/docs/react-without-jsx.md [hunk 6]
```diff
@@ -37,12 +37,11 @@ ReactDOM.render(
...
 
-If you're curious to see more examples of how JSX is converted to JavaScript, you can try out [the online Babel compiler](babel://jsx-simple-example).
+A komponens megadható sztringként, a `React.Component` alosztályaként vagy egy sima függvényként.
 
-The component can either be provided as a string, as a subclass of `React.Component`, or a plain function.
-
-If you get tired of typing `React.createElement` so much, one common pattern is to assign a shorthand:
+Ha úgy érzed, hogy túl időigényes mindig kigépelni a `React.createElement`et, egy gyakori megoldás, a függvény egy rövidítéshez rendelése:
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:44]:
```suggestion
Ha úgy érzed, hogy túl időigényes mindig kigépelni a `React.createElement`-et, egy gyakori megoldás, a függvény egy rövidítéshez rendelése:
```

File: content/docs/react-without-jsx.md [hunk 7]
```diff
@@ -52,8 +51,7 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
+A `React.createElement`nek ezt a rövidített formáját használva, már sokkal kényelmesebben React kódot írni, még JSX nélkül is.
```
Comment (balazsorban44) [inline content/docs/react-without-jsx.md:54]:
```suggestion
A `React.createElement`-nek ezt a rövidített formáját használva, már sokkal kényelmesebb React kódot írni, még JSX nélkül is.
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

File: content/docs/react-without-jsx.md (hunk 5 above)
Comment (RichardOtvos) [reply content/docs/react-without-jsx.md:40]: Sure!

**Review by balazsorban44 — APPROVED** [review]
Nice work, thank you! :slightly_smiling_face:

**Pull request merged**